src/centos/scripts/start-marklogic.sh

```diff
@@ -28,13 +28,13 @@ for var in OVERWRITE_ML_CONF INSTALL_CONVERTERS MARKLOGIC_DEV_BUILD MARKLOGIC_IN
     declare $var="$(echo "${!var}" | sed -e 's/[[:blank:]]//g' | awk '{print tolower($0)}')"
 done
 
-# define log and err functions
-log() {
-    echo "$(basename "${0}"): ${*}"
-}
-err() {
-    echo "$(basename "${0}") ERROR: ${*}" >&2
-    exit 1
+#logit utility function
```

Comment:
I'm not personally a fan of a function like this where the number is the level. It is hard to read in the code as well as write in the code because you have to always reference back to make sure you have the "level" number correct. I would recommend separate functions: info(), debug(), error(). You could make a generic log() function which each of those call if you want.

Also, the error() function could accept an argument of whether it should exit or not.

Reply:
Updated the log function, added info() and error() fn